src/Store/api/rest/index.ts

```diff
@@ -17,12 +17,14 @@ import {
   RestFailure,
   RestStart,
   ApiRequest,
+  StatsSuccess,
 } from "Store/api/rest/types";
 import { Errors } from "io-ts";
 
 export const restDispatch = createAction<RestDispatch>("api/restDispatch");
 export const restStart = createAction<RestStart>("api/restStart");
 export const restSuccess = createAction<RestSuccess>("api/restSuccess");
+export const statsSuccess = createAction<StatsSuccess>("api/statsSuccess");
```

Comment:
this shouldn't go here; it should go next to the route definition (or maybe in a separate file);
this file is for the hidden rest API implementation middleware and associated routes